api/NamedNodeMap.json

```diff
@@ -218,10 +218,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/NamedNodeMap/length",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "44"
```

Comment:
I don't think this is right, it's an attribute on prototype problem. http://mdn-bcd-collector.appspot.com/tests/api/NamedNodeMap might need a custom test to fix this.